src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -281,3 +281,42 @@ def prepare_logs_argument(c):
                    arg_type=get_three_state_flag(),
                    help="Disable Application Insights.",
                    validator=validate_app_insights_parameters)
+
+    with self.argument_context('spring-cloud application-configuration-service bind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    with self.argument_context('spring-cloud application-configuration-service unbind') as c:
+        c.argument('app', app_name_type, help='Name of app.', validator=validate_app_name)
+
+    with self.argument_context('spring-cloud application-configuration-service git repo add') as c:
```

Comment:
use a loop to wrap add and update? see `spring-cloud create` and `spring-cloud update` for example